src/MOI_wrapper/MOI_wrapper.jl

```diff
@@ -661,7 +662,13 @@ function MOI.get(model::Optimizer, ::MOI.ListOfVariableAttributesSet)
 end
 
 function MOI.get(model::Optimizer, ::MOI.ListOfModelAttributesSet)
-    attributes = Any[MOI.ObjectiveSense()]
+    if MOI.is_empty(model)
+        return Any[]
+    end
+    attributes = Any[]
+    if model.objective_sense !== nothing
+        push!(attributes, MOI.ObjectiveSense())
+    end
     typ = MOI.get(model, MOI.ObjectiveFunctionType())
     if typ !== nothing
```

Comment:
Is this passing tests? It should be
```Julia
if model.is_objective_set
    F = MOI.get(model, MOI.ObjectiveFunctionType())
    push!(attributes, MOI.ObjectiveFunction{F}())
end
```
(ObjectiveFunctionType now never returns `nothing`.)

Reply:
Yes, tests passed even if this is not corrected. Maybe a new MOI test should be added?